packages/components/src/date-picker/DatePicker.module.css

```diff
@@ -50,13 +50,12 @@
   .calendarButton {
     border: none;
     margin-left: auto;
-    height: --size-150;
+    height: calc(--size-150 - 1px);
```

Comment:
```suggestion
    height: 100%;
```

Det såg lite funkigt ut när jag zoomade ut:

![Skärmbild 2025-06-03 075310](https://github.com/user-attachments/assets/8df41d35-c745-41fc-9c5d-0c2f23aa6164)

`height: 100%`  verkar fungera ännu bättre:

![Skärmbild 2025-06-03 075317](https://github.com/user-attachments/assets/a73e931a-9406-4462-a0d3-3d2642c9ccaa)